composer.json

```diff
@@ -37,6 +37,8 @@
         "mockery/mockery": "^1.0",
         "moontoast/math": "^1.1",
         "php-mock/php-mock-phpunit": "^2.0",
+        "phpstan/phpstan-phpunit": "0.9.2",
+        "phpstan/phpstan-shim": "0.9.1",
```

Comment:
this is the actual tool, but wrapped in a phar so it does not install any other dependencies